src/xenia/kernel/xboxkrnl/xboxkrnl_crypt.cc

```diff
@@ -7,12 +7,18 @@
 ******************************************************************************
 */
 
+#include <algorithm>
+
 #include "xenia/base/logging.h"
 #include "xenia/kernel/kernel_state.h"
 #include "xenia/kernel/util/shim_utils.h"
 #include "xenia/kernel/xboxkrnl/xboxkrnl_private.h"
 #include "xenia/xbox.h"
 
+#ifdef XE_PLATFORM_WIN32
```

Comment:
xenia/base/platform.h should be included for this definition, as there are no other includes in this file that clearly imply it being included.